drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c

```diff
@@ -4000,10 +3995,10 @@ static struct platform_driver bcm2835_v4l2_codec_driver = {
 		   },
 };
 
-module_platform_driver(bcm2835_v4l2_codec_driver);
+module_vchiq_driver(bcm2835_v4l2_codec_driver);
 
 MODULE_DESCRIPTION("BCM2835 codec V4L2 driver");
 MODULE_AUTHOR("Dave Stevenson, <dave.stevenson@raspberrypi.com>");
 MODULE_LICENSE("GPL");
 MODULE_VERSION("0.0.1");
-MODULE_ALIAS("platform:bcm2835-codec");
+MODULE_ALIAS("bcm2835-codec");
```

Comment:
yes, this should be:
`MODULE_ALIAS("vchiq:bcm2835-codec");`
Testing ...
```
kbingham@mars:~ $ lsv4l2 
v4l-subdev0: unicam
v4l-subdev1: imx219 10-0010
video0: unicam-image
video1: unicam-embedded
video10: bcm2835-codec-decode
video11: bcm2835-codec-encode
video12: bcm2835-codec-isp
video13: bcm2835-isp-output0
video14: bcm2835-isp-capture0
video15: bcm2835-isp-capture1
video16: bcm2835-isp-stats2
video18: bcm2835-codec-image_fx
video19: rpivid
video20: bcm2835-isp-output0
video21: bcm2835-isp-capture0
video22: bcm2835-isp-capture1
video23: bcm2835-isp-stats2
video31: bcm2835-codec-encode_image
```

I'll rebase, fixup and repush